ui/v2.5/src/components/Shared/DurationInput.tsx

```diff
@@ -93,7 +93,7 @@ export const DurationInput: React.FC<IProps> = (props: IProps) => {
   }
 
   return (
-    <Form.Group className={`duration-input ${props.className}`}>
+    <div className={`duration-input ${props.className}`}>
```

Comment:
This way we avoid adding the `form-group` class, which adds some margin at the bottom and thus makes all the forms containing `DurationInput` look inconsistent with the rest of the form inputs. Check the input:
1. Settings -> Interface -> Maximum loop duration
2. Settings -> DLNA -> Enable temporarily
3. Movies -> New -> Duration
4. Scenes -> Filter -> Duration